compiler/rustc_resolve/src/imports.rs

```diff
@@ -718,16 +739,18 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
             PathResult::Indeterminate => unreachable!(),
         };
 
-        let (ident, target, source_bindings, target_bindings, type_ns_only) = match import.kind {
+        let (ident, target, source_bindings, target_bindings, type_ns_only, id) = match import.kind
```

Comment:
Nit: `import_id` would be a bit more descriptive. There are many `id`s around here.